tutorials/data-science-exploration/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Exploratory queries with Google BigQuery
+description: Use BigQuery to investigate arbitrarily large datasets.
+author: jerjou
+tags: Data Science, BigQuery
+date_published: 2017-05-10
+---
+[Google BigQuery](/bigquery) is a fully-managed service for data
+exploration using an SQL syntax. Once data has been loaded into the service, it
+can be queried at interactive speeds, without having to know beforehand what
+you're looking for.
+
+There has been much already written about using BigQuery for data analysis - for
+example:
+
+* [Analyzing Financial Time Series using BigQuery](/solutions/time-series/bigquery-financial-forex)
+* [How to forecast demand with Google BigQuery, public datasets and TensorFlow](/blog/big-data/2016/05/how-to-forecast-demand-with-google-bigquery-public-datasets-and-tensorflow)
+* Any of the articles around the [BigQuery Public Datasets](/bigquery/public-data/)
+
+For the purposes of this tutorial, we'll focus on finding meaning in the data
+that we've ingested from [previous](preprocessing.md) [articles](extraction.md).
```

Comment:
Also, though it isn't a show-stopper by any means, it would be better if this tutorial were written in the second person imperative to provide a more active tone. Meaning, where you have written, "We did a thing," change it to "You did a thing". And wherever possible change, "We will do x," to, "Do x," or, "Next you do x".